### PR TITLE
Merge Documents to Develop Branch

### PR DESCRIPTION
**Pull request by iwanlevin**
Merge this week's documentation to the working branch of our repository

### CODE REVIEW

File: docs/week14_scrum.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## 2. Learning Activity
+
+N/A
+
+## 3. Plan
+
+**Task for this week:**
+
+- Fix malformed buffer copy
```
Comment (cppcooper) [inline docs/week14_scrum.md:27]: according to the template this should have a name associated

File: docs/week15_scrum.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - Explored benefits over undocumented Orthanc SDK functions
+  - Explored cost of implementation
+  - Explored performance and scaling
+
+## 3. Plan
+
+**Task for this week:**
+
+- Start implementing ODBC & crosswalk table
+- Start building frontend
```
Comment (cppcooper) [inline docs/week15_scrum.md:33]: same thing, the tasks/activities are supposed to have names associated

**Review by cppcooper — COMMENTED** [review]
Nearly there

File: docs/week15_scrum.md (hunk 2 above)
Comment (brianzhouzc) [reply docs/week15_scrum.md]: Added

File: docs/week14_scrum.md (hunk 1 above)
Comment (brianzhouzc) [reply docs/week14_scrum.md]: Added

**Review by brianzhouzc — APPROVED** [review]
Looks good to me

**Review by cppcooper — APPROVED** [review]
This seems to have everything I'm privy to.

**Pull request merged**